wallets/provider-sui-wallet/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "@rango-dev/provider-sui-wallet",
+  "version": "1.0.0",
+  "license": "MIT",
+  "type": "module",
+  "source": "./src/mod.ts",
+  "main": "./dist/mod.js",
+  "exports": {
+    ".": "./dist/mod.js"
+  },
+  "typings": "dist/mod.d.ts",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "scripts": {
+    "build": "node ../../scripts/build/command.mjs --path wallets/provider-sui-wallet --inputs src/mod.ts",
+    "ts-check": "tsc --declaration --emitDeclarationOnly -p ./tsconfig.json",
+    "clean": "rimraf dist",
+    "format": "prettier --write '{.,src}/**/*.{ts,tsx}'",
+    "lint": "eslint \"**/*.{ts,tsx}\""
+  },
+  "dependencies": {
+    "@rango-dev/wallets-shared": "^0.43.1-next.2",
+    "@rango-dev/signer-sui": "^0.2.1-next.0",
+    "@mysten/sui": "^1.21.2",
+    "@mysten/wallet-standard": "^0.13.26",
+    "rango-types": "^0.1.81"
```

```suggestion
    "rango-types": "^0.1.82"
```
Also please check that `yarn.lock` should not contain any changes.

---

Done